docs/releases/readme.md

```diff
@@ -45,6 +45,10 @@ _Outcome_: **Team is aware of release and in agreement about what fixes & featur
     -   For _major_ and _minor_ releases, create branch: `release/X.X`.
     -   For _patch_ releases, the branch should already exist.
 
+#### Create a release pull request
```

Comment:
This should clarify which versions need this template - patches do not need this level of scuitiny.

Reply:
I disagree, I think we should be consistent in all our releases to use the same process. The checks will be more straightforward and less notes but since we are still going to be creating tags for these patches and then a pull request in WC core to bump the tag, I think copying the completed description will work a lot better for that (and I want to automate that as well).

Reply:
Thats a lot of work for a patch release which will interfere with other cycle work. Look at 2.6.1 - there was a single change there. Are you suggesting we need a throwaway PR to communicate something so small?

Reply:
> Are you suggesting we need a throwaway PR to communicate something so small?

I don't think it's a throw-away PR since the notes from the PR will be copied into the pull request bumping the tag for WC core. I also disagree that it's a lot of work. More work? yes. At what point do we decide there are enough changes in a patch release to warrant additional notes? 

Another important thing, from the perspective of our team, these PR's can seem redundant, but they aren't done _primarily_ for our benefit (although I do think there is benefit for us too). Primarily this process is done for the benefit of the WooCommerce core team and the pull request we create in WooCommerce core for merging the changes. Once it becomes a part of our workflow (and hopefully automated soon), the extra time cost I think is worth it when it comes to consistent communication and process around our releases.

Viewed through the lense of the 15 patch releases we had for 2.5.14 I can certainly understand your sentiment on the cost, but viewed through the lense of moving back to _bi-weekly_ releases, I don't anticipate we'll have that many patch releases that make this a significant chore.

Reply:
> I don't think it's a throw-away PR since the notes from the PR will be copied into the pull request bumping the tag for WC core

Noting we also need to update:
- The release tag description.
- The dev blog.
- Possibly p2s.

Even if thats duplication and can be copy/pasted, if notes need updating that is a lot of places to keep in sync which is part of my concern. The other part is the large checklist in the PR template, which I get is somewhat optional, but it's still there.

> At what point do we decide there are enough changes in a patch release to warrant additional notes?

If our changelog entries for patch release fixes are descriptive enough, the need for supplemental notes should be minimal fwiw. I understand supplemental content is useful for new features but that doesn't apply here.

Reply:
Your concerns are noted, but I still think having the checklist consistently applied removes a roadblock in terms of "Does this release need more things noted? Do I have everything covered? Are the appropriate places updated?" For patch releases there definitely will be cases where whoever is doing the release just checks the items and has no additional notes to add - but once this is automated, I don't think it will be that much of a chore. As I implement the automation I can look at pulling in the generated changelog directly into the pull request description for patch releases to eliminate that portion of the notes.

> Even if thats duplication and can be copy/pasted, if notes need updating that is a lot of places to keep in sync which is part of my concern.

I would agree if this is something that needs kept in sync over a period of time, but for a release, that's just a one time thing right? So copy/paste should be sufficient where its needed (and copy/paste is a good signal of what can be automated as well - for instance release tag description and maybe eventually even dev blog post via WP REST API).

The purpose of the manual work _initially_ is to help both refine what works and doesn't work, and also identify what could be automated (which I'm pretty keen on implementing asap).

While I think your concerns are important, I'd still like to try this and see how it works.